chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -101,9 +96,9 @@ func New(node *neutrino.ChainService) (*NeutrinoNotifier, error) {
 
 		rescanErr: make(chan error),
 
-		newBlocksUpdateSignal: make(chan struct{}),
+		newBlocks: NewConcurrentQueue(),
```

Comment:
Here new blocks and stale blocks are being handled in parallel. I wonder if we should keep the ordering (making sure stale blocks get handled before the new blocks during a reorg), similar to what was done in #364?

Reply:
Definitely agree, and I'm looking into that, but I think it should be done as a separate PR.